src/main/java/app/service/impl/FileReadServiceImplementation.java

```diff
@@ -0,0 +1,28 @@
+package app.service.impl;
+
+import app.service.FileReadService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileReadServiceImplementation implements FileReadService {
+    public static final String SEPARATOR_TO_SPLIT = ",";
+
+    @Override
+    public List<List<String>> readFile(String filePath) {
+        String line;
+        List<List<String>> dataFromFile = new ArrayList<>();
```

Comment:
```suggestion
        List<Transaction/Record> dataFromFile = new ArrayList<>();
```
it will be better to operate with specific model with fields type, fruitName, quantity etc than raw line